### DISCUSSION TITLE
Generate Plex collections based on user requests

### DISCUSSION THREAD

**Discussion by marcodejongh** [sct/overseerr#3435] [Feature Requests]
I was playing around with Plex Meta Manager(PMM) for generating collections, and realised it's a bit overkill for my relatively small media collection. However, the idea of creating collections for my users of the content they requested themselves does seem very useful to me. 

Since this is basically a very small part of what PMM can do, I think it's reasonable to build this feature separately into overseerr itself rather than contributing back to PMM. Also, since Overseerr already has all the information available anyway. Where-as PMM currently has no knowledge at all of Overseerr.

**Comment by samohtxotom** [comment]
Should have a PR for this in the coming days
